packages/docs/src/components/examples/Example.vue

```diff
@@ -65,6 +65,7 @@
                 <markup
                   :code="pen[section]"
                   :rounded="false"
+                  :language="section === 'template-pug' ? 'pug' : undefined"
```

Comment:
I don't think this is the best way to go (but it works ...)
Can someone help me improve this / guide me?